api/integrations/github/tasks.py

```diff
@@ -85,10 +76,15 @@ def generate_feature_external_resources(
             for resource in feature_external_resources
         ]
 
-    if github_event_data.type == WebhookEventType.FLAG_DELETED.value:
+    if (
+        github_event_data.type == WebhookEventType.FLAG_DELETED.value
+        or github_event_data.type == WebhookEventType.SEGMENT_OVERRIDE_DELETED.value
+    ):
         feature_external_resources = generate_feature_external_resources(
-            FeatureExternalResource.objects.filter(
-                feature_id=github_event_data.feature_id
+            list(
+                FeatureExternalResource.objects.filter(
+                    feature_id=github_event_data.feature_id
+                ).all()
             )

```

Comment:
I think a cast to `list` is enough:
```suggestion
            list(
                FeatureExternalResource.objects.filter(
                    feature_id=github_event_data.feature_id
                )
            )
```

Reply:
Done